contribs/gmf/apps/mobile/partials/mobilegeolocation.html

```diff
@@ -0,0 +1,4 @@
+<button
+  ng-click="ctrl.toggleTracking()"
+  class="gmf-mobile-geolocation">
+</button>
```

Comment:
This should be outside `gmf/apps` ...
The partials will not be related on the directive ?
@fgravin ?